src/test/c_api/config_standard_dummy.json

```diff
@@ -3,6 +3,25 @@
         {"config": {
                 "name": "dummy",
                 "base_path": "/ovms/src/test/dummy",
-                "shape": "(1, 10)"}}
+                "plugin_config": {
```

Comment:
This started to become not so _standard_ dummy. Use yet another config.json file, named "config_dummy_benchmark.json"?